core/internal/mocks/go_generate.go

```diff
@@ -0,0 +1,6 @@
+package mocks
+
+//go:generate mockery --srcpkg github.com/smartcontractkit/chainlink/core/gethwrappers/generated/flux_aggregator_wrapper --name FluxAggregatorInterface --output . --case=underscore --structname FluxAggregator --filename flux_aggregator.go
```

Comment:
Note I added this to cleanly separate the mockery gens from the gethwrapper gens. These used to be in gethwrappers/go_generate.go which didn't make a lot of sense